es6-classes/example-class.js

```diff
@@ -0,0 +1,19 @@
+class ExampleClass {
+  // eslint-disable-next-line no-useless-constructor
+  constructor() {
+
+  }
```

Comment:
If there isn't anything in a `constructor` we leave out the `constructor`. The `ExampleClass` should be empty.